artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/BrokerConnectionControlImpl.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.management.impl;
+
+import javax.management.MBeanAttributeInfo;
+import javax.management.MBeanOperationInfo;
+import javax.management.NotCompliantMBeanException;
+
+import org.apache.activemq.artemis.api.core.management.BrokerConnectionControl;
+import org.apache.activemq.artemis.core.persistence.StorageManager;
+import org.apache.activemq.artemis.core.server.BrokerConnection;
+import org.apache.activemq.artemis.logs.AuditLogger;
+
+public class BrokerConnectionControlImpl extends AbstractControl implements BrokerConnectionControl {
+
+   private final BrokerConnection brokerConnection;
+
+   public BrokerConnectionControlImpl(BrokerConnection brokerConnection,
+                                      StorageManager storageManager) throws NotCompliantMBeanException {
+      super(BrokerConnectionControl.class, storageManager);
+      this.brokerConnection = brokerConnection;
+   }
+
+   @Override
+   public boolean isStarted() {
+      if (AuditLogger.isBaseLoggingEnabled()) {
+         AuditLogger.isStarted(this.brokerConnection);
+      }
+      clearIO();
+      try {
+         return brokerConnection.isStarted();
```

Comment:
The mismatch of "this.brokerConnection" and then just "brokerConnection" seems weird when reading, be nicer if consistent.

(here and below)